pius

```diff
@@ -88,10 +88,13 @@ def check_options(parser, options, args):
         print("NOTE: -O and -m are present, turning on -e")
         options.encrypt_outfiles = True
 
-    if options.mail_user and not options.mail_tls:
-        print("NOTE: -u is present, turning off -S.")
+    if options.mail_user and not (options.mail_tls or options.mail_ssl):
+        print("NOTE: -u is present, enabling TLS by default.")
```

Comment:
we already enable TLS by default, so this probably means someone disabled it, so this wording is slightly confusing (though not as confusing as the old incorrect text, lol)

```suggestion
        print("NOTE: -u (SMTPAUTH) is present, enabling TLS. Either TLS or SSL is required with SMTPAUTH")
```